### ISSUE TITLE
Add syntax highlighting option.

### ISSUE THREAD

**Issue by johnotander** [johnotander/ember-remarkable#2]

**Comment by srsgores** [comment]
+1.  What is the progress on this?

**Comment by johnotander** [comment]
Whoops, this issue should've been closed a while ago! It should be on by default now.

**Comment by johnotander** [comment]
You can see https://github.com/markdowncss/splendor/blob/master/gulpfile.js for an example of this plugin in action.

**Comment by johnotander** [comment]
Oh man am I dumb, thought this was my `gulp-remarkable` plugin. My apologies.

**Comment by johnotander** [comment]
I've added this to my list to attempt to tackle tomorrow, I will let you know how it goes @srsgores

**Comment by srsgores** [comment]
@johnotander, thanks.  I just had a look into it, and it appears that we might need [highlightjs](https://github.com/isagalaev/highlight.js) in order to get things working (at least the styles for that).  And it looks like [the option we want to set it is``highlight``](https://github.com/jonschlinkert/remarkable#user-content-syntax-highlighting).

**Comment by johnotander** [comment]
Yep, @srsgores you're exactly right, I've just added a highlight.js shim and bower dependency.

It's basically together now, implemented with some rudimentary tests. Tomorrow I will add documentation, bump the version, verify everything is working in a test app, and publish to npm (v0.2.0).

If you want to take it for a spin, feel free to install from the `add-highlightjs-support` branch, and let me know how it goes: https://github.com/johnotander/ember-remarkable/tree/add-highlightjs-support

Something similar to the following should illustrate whether it's working or not:

```hbs
{{md-text text='# Markdown is fun\n ```js\nvar awesome = require("awesome");```'}}
{{md-text text='# Markdown is fun\n ```var awesome = require("awesome");```'}}
```

:)

**Comment by johnotander** [comment]
The relevant PR: https://github.com/johnotander/ember-remarkable/pull/4

**Comment by srsgores** [comment]
@johnotander, looks good.  I'll try to have a go at it tomorrow.

Thanks for your prompt responses.

**Comment by johnotander** [comment]
Published to npm as `ember-remarkable@1.0.0`.

**Comment by srsgores** [comment]
@johnotander, did you forget to include ``highlightjs`` in the ``dependencies`` section of ``package.json``?

> Path or pattern "bower_components/highlightjs/highlight.pack.js" did not match any files
Error: Path or pattern "bower_components/highlightjs/highlight.pack.js" did not match any files
  at Object.multiGlob (C:\Users\_admin\Documents\GitHub\seangoresht\node_modules\ember-cli\node_modules\broccoli-kitchen-sink-helpers\index.js:202:13)
  at C:\Users\_admin\Documents\GitHub\seangoresht\node_modules\ember-cli\node_modules\broccoli-concat\index.js:62:32
  at $$$internal$$tryCatch (C:\Users\_admin\Documents\GitHub\seangoresht\node_modules\ember-cli\node_modules\rsvp\dist\rsvp.js:470:16)
  at $$$internal$$invokeCallback (C:\Users\_admin\Documents\GitHub\seangoresht\node_modules\ember-cli\node_modules\rsvp\dist\rsvp.js:482:17)
  at $$$internal$$publish (C:\Users\_admin\Documents\GitHub\seangoresht\node_modules\ember-cli\node_modules\rsvp\dist\rsvp.js:453:11)
  at $$rsvp$asap$$flush (C:\Users\_admin\Documents\GitHub\seangoresht\node_modules\ember-cli\node_modules\rsvp\dist\rsvp.js:1531:9)
  at process._tickCallback (node.js:419:13)

**Comment by srsgores** [comment]
It also appears that the assets for highlightjs aren't being included...

**Comment by johnotander** [comment]
Hrm, did you run `ember g ember-remarkable` after installing the package?

**Comment by srsgores** [comment]
@johnotander, yes.

**Comment by johnotander** [comment]
Okay, thanks for checking. I'll get digging here shortly.

**Comment by johnotander** [comment]
Fixed the problem (wasn't specifying both bower dependencies correctly). Published as `ember-remarkable@1.0.1`.

Sorry about that, should be in working order now!
